packages/libraries/core/package.json

```diff
@@ -43,7 +43,8 @@
     "@fluentui-react-native/radio-group": "0.16.33",
     "@fluentui-react-native/separator": "0.14.14",
     "@fluentui-react-native/tabs": "0.11.31",
-    "@fluentui-react-native/text": "0.19.19"
+    "@fluentui-react-native/text": "0.19.19",
+    "@fluentui-react-native/spinner": "^0.1.1"
```

Comment:
I do not think we need to add this as part of core library - my understanding is that we are discouraging adding more components here in favour of explicit imports by partners.

Reply:
removed.